front/src/applications/operationalStudies/v2/components/ScenarioHeader.tsx

```diff
@@ -170,7 +170,13 @@ const ScenarioHeader = ({
         {/* user informations */}
         <div className="user-info">
           <div className="spacer" />
+          {impersonatedUser && <Hubot size="lg" className="mr-2 text-black" />}
           <UserActionsDropdown className="dropdwon-position" titleContent={userDropdownTitle} />
+          {impersonatedUser && (
+            <button type="button" onClick={logout}>
+              <SignOut className="ml-2 text-black" />
```

Comment:
```suggestion
              <SignOut className="ml-3 text-black" />
```
To have the same spacing as on the sketch